src/app/routes/wizard/wizard.component.ts

```diff
@@ -48,7 +50,8 @@ export class WizardComponent {
     this.store.dispatch(new Settings.SetDisplayRateAction({ value, prev }));
   }
 
-  createItemObjective(targetId: string): void {
+  async createItemObjective(targetId: string): Promise<void> {
+    await this.router.navigate(['list']);
```

Comment:
Yeah, it would start navigating, but before it finished, the addition of a new objective would cause another navigation to start (adding the objective to the url) and that would essentially cancel out the in-progress navigation. 
Now it will wait until it's moved to the list page before adding the objective, ensuring that when the second navigation happens it stays on the right page.

Reply:
My guess is this doesn't occur if you've already loaded the list page before, because in that case the navigation to the list page is fast enough to avoid being cancelled out